### DISCUSSION TITLE
Any free LV2 for Windows around?

### DISCUSSION THREAD

**Discussion by vcvrackideas** [DISTRHO/Cardinal#61] [Q&A]
I found one recently and it was found correctly from common files/lv2 folder which is perfectly cool but maybe there is some list around with Windows dll? I know it's more for Linux but I like how it works right inside the module loader you msde

**Comment by falkTX** [comment]
You have some in https://github.com/DISTRHO/PawPaw/releases and https://x42-plugins.com/x42/ provides a good collection too.
Eventually Ildaeil will be made to load VST plugins too.

**Comment by falkTX** [comment]
btw, since these are external resources cardinal "factory" presets are not allowed to use them, because not everyone is going to have them installed.

I have not yet fully decided what to do with the Ildaeil project, for now it is mostly a test of the host features of Carla (which it uses behind the scenes).

**Comment by falkTX** [comment]
Moving this ticket as a discussion, since it is not really an issue but more of a question.
